packages/google_sign_in/google_sign_in_platform_interface/lib/src/utils.dart

```diff
@@ -5,23 +5,20 @@
 import '../google_sign_in_platform_interface.dart';
 
 /// Converts user data coming from native code into the proper platform interface type.
-GoogleSignInUserData getUserDataFromMap(Map<String, dynamic> data) {
+GoogleSignInUserData? getUserDataFromMap(Map<String, dynamic>? data) {
   if (data == null) {
```

Comment:
I think this should throw since it's used in an async chain

Reply:
I did make that change for token data... I am less sure about user data. I did a sweep of the android/ios code and saw that they never return null user map but there's a test for it:

```
    test('can sign in after aborted flow', () async {
      responses['signIn'] = null;
      expect(await googleSignIn.signIn(), isNull);
      responses['signIn'] = kUserData;
      expect(await googleSignIn.signIn(), isNotNull);
    });
```

Reply:
got it. Yeah, it seems tricky to change.